internal/engine/client.go

```diff
@@ -93,7 +93,7 @@ func waitBuildkit(ctx context.Context, host string) ([]*bkclient.WorkerInfo, err
 	// it needs to synchronize cache state.
 	const (
 		retryPeriod   = 100 * time.Millisecond
-		retryAttempts = 1800
+		retryAttempts = 6000
```

Comment:
I know that we want this in today, which we will do, but it would be nice to extract these into configurable properties such as flags **and** environment variables ideally.

Want to do a follow-up @jlongtine @sipsma?

Reply:
Sounds good, @gerhard!